ccd-client/src/main/java/uk/gov/hmcts/cmc/ccd/client/service/StartCase.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.hmcts.cmc.ccd.client.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+import uk.gov.hmcts.cmc.ccd.client.header.HttpHeadersFactory;
+import uk.gov.hmcts.cmc.ccd.client.model.EventRequestData;
+import uk.gov.hmcts.cmc.ccd.client.model.StartEventResponse;
+
+@Service
+public class StartCase {
+
+    private final RestTemplate restTemplate;
+    private final HttpHeadersFactory headersFactory;
+
+    @Value("${ccd.caseDataStore.baseUrl}")
+    private String ccdUrl;
+
+    @Autowired
+    public StartCase(final RestTemplate restTemplate, final HttpHeadersFactory httpHeadersFactory) {
+        this.restTemplate = restTemplate;
+        this.headersFactory = httpHeadersFactory;
+    }
+
+    public StartEventResponse exchange(final EventRequestData eventRequestData) {
+
+        final HttpEntity<String> httpEntity = new HttpEntity<String>(headersFactory.getHttpHeader());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS2293)